tests/functional/test_submitted_charts.py

```diff
@@ -326,7 +326,7 @@ def submission_tests_run_for_submitted_charts(secrets):
 
             # Create PR from test_repo:pr_branch to test_repo:base_branch
             data = {'head': pr_branch, 'base': base_branch,
-                    'title': pr_branch}
+                    'title': pr_branch, 'body': os.environ.get('PR_BODY')}
```

Comment:
Good candidate to move to utils.

Reply:
I would prefer leave this message body here because each github post request may have different request bodies.. the bigger chunk for example creation of temporary branches can be refactored to utils (in the future improvements)